spec/shop_spec.rb

```diff
@@ -4,14 +4,22 @@
 describe Shop do
   it 'responds to name' do #description of your spec/test
     shop = Shop.new
-
     expect(shop).to respond_to(:name)
     #expects shop instance to have an name accessor/method
   end
 
-  it 'sums two numbers' do
-    result = Shop.sum(3,4)
+describe "#create" do
```

Comment:
indentation. the describe blocks for your methods should be nested within the describe block of your class